scripts/setup-eclipse.sh

```diff
@@ -102,3 +102,6 @@ cat >> $classpath_file << 'EOF'
   <classpathentry kind="output" path="target/classes"/>
 </classpath>
 EOF
+
+echo "Set Java code formatter file in Eclipse as following steps:"
+echo "Properties –> Java Code Style –> Formatter –> Enable project specific settings and load 'scripts/resources/eclipse/JavaCodeFormatter.xml'"
```

Comment:
Can this not be set by the script somehow, maybe in the project file. IntelliJ supports something like that.